adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h

```diff
@@ -0,0 +1,29 @@
+//
+//  VungleAdNetworkExtras.h
+//
+//  Copyright © 2016 Vungle. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <GoogleMobileAds/GADAdNetworkExtras.h>
+
+@interface VungleAdNetworkExtras : NSObject<GADAdNetworkExtras>
+
+/*!
+ * @brief NSString with user identifier that will be passed if the ad is incentivized.
+ * @discussion Optional. The value passed as 'user' in the an incentivized server-to-server call.
+ */
+@property (nonatomic, copy) NSString * _Nullable userId;
+
+/*!
+ * @brief Controls whether presented ads will start in a muted state or not.
+ */
+@property (nonatomic, assign) BOOL muted;
+
+
+@property (nonatomic, copy) NSArray<NSString *> * _Nonnull placements;
```

Comment:
Please use a more descriptive name for this property. You can use `allPlacements` similar to the [Android adapter](https://github.com/Vungle/googleads-mobile-android-mediation/blob/f65ea147a484fdf42c51a4c44b9f9d6bdb619bd0/ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java#L13).